### PR TITLE
Add ophan tracking to external video embeds

### PR DESCRIPTION
**Pull request by phamann**
This PR does two tightly coupled things:
- Add Ophan tracking for our external video player embeds
- Add new configuration value to point to new tracker-js file `ophan/embed`
- Refactor and abstract the media player event system into a separate module so that it can be shared by both versions of the player, and in-turn making it more dry.

/cc @rich-nguyen @mattosborn

### CODE REVIEW

**Comment by phamann** [comment]
I have tested extensively that this doesn't cause any breaking changes or regressions on normal video player tracking.

File: static/src/javascripts/projects/common/modules/analytics/omnitureMedia.js [hunk 1]
```diff
@@ -19,6 +21,7 @@ define([
             mediaType = qwery('audio', player.el()).length ? 'audio' : 'video',
             contentStarted = false,
             prerollPlayed = false,
+            isEmbed = !!guardian.isEmbed,
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/analytics/omnitureMedia.js]: `isEmbed` appears to be used once, inline?

**Comment by rich-nguyen** [comment]
:+1:

**Comment by phamann** [comment]
@rich-nguyen Thinking about this, a bit more. I also want to add a switch for `embedTracking` so I can ship this before ophan and then test again on code etc.

**Pull request merged**